draft-ietf-taps-interface.md

```diff
@@ -460,11 +460,13 @@ earlier stages and querying them in later stages:
 - Message Properties can be set on Preconnections and Connections
 - The effect of Selection Properties can be queried on Connections and Messages
 
-Note that Configuring Connection Properties and Message Properties on
+Note that configuring Connection Properties and Message Properties on
 Preconnections is preferred over setting them later. Early specification of
 Connection Properties allows their use as additional input to the selection
-process. Protocol Specific Properties, see {{property-names}}, should not be
-used as an input to the selection process.
+process. Protocol Specific Properties, that enable configuration of Specialized
```

Comment:
I think it should be "which enable" instead of "that enable" here, because this adds information Protocol Specific Properties (explaining what they are) rather than narrowing down the set of Protocol Specific Properties.

Reply:
Done